spec/faked_project/lib/faked_project/some_class.rb

```diff
@@ -11,11 +11,9 @@ def reverse
   end
 
   def compare_with(item)
-    if item == label
-      return true
-    else
-      fail "Item does not match label"
-    end
+    return true if item == label
```

Comment:
I know this is not applied to all of the code base and this especially is "just" our test sample code, but personally I prefer the actual if/else form because it's easier to parse the control flow of the method